agentops/worker.py

```diff
@@ -42,7 +42,7 @@ def flush_queue(self) -> None:
                                 self.config.api_key,
                                 self.config.parent_key)
 
-    def start_session(self, session: Session) -> None:
+    def start_session(self, session: Session) -> bool:
```

Comment:
nothing is returned in this function

Reply:
In `client.py` the class `Client` defines the `start_session` function as - 

```python
def start_session(self, tags: Optional[List[str]] = None, config: Optional[Configuration] = None):
    """
    Start a new session for recording events.

    Args:
        tags (List[str], optional): Tags that can be used for grouping or sorting later.
            e.g. ["test_run"].
        config: (Configuration, optional): Client configuration object
    """
    if self._session is not None:
        return logging.warning("🖇 AgentOps: Cannot start session - session already started")

    if not config and not self.config:
        return logging.warning("🖇 AgentOps: Cannot start session - missing configuration")

    self._session = Session(uuid4(), tags or self._tags, host_env=get_host_env())
    self._worker = Worker(config or self.config)
    start_session_result = self._worker.start_session(self._session)
    if not start_session_result:
        self._session = None
        return logging.warning("🖇 AgentOps: Cannot start session")
```

`start_session_result` is expecting the function to return something and there are return statements for the same.

How is it not returning anything then?

Reply:
Ah, I was mistaking `worker.start_session()` for `client.start_session()`